### ISSUE TITLE
AddComment() silently fails in 5.6.0 and above

### ISSUE THREAD

**Issue by johndoh** [EPPlusSoftware/EPPlus#387]
With version 5.6.0 and above comments attached to cells are no longer visible in the generated file. With the same code comments are visible (small red triangle top right corner) in the file.

```sheet.Cells[1, 1].AddComment("My Comment", "Me")```

**Comment by swmal** [comment]
Hi,
thanks for reporting, we will have a look at this.

**Comment by johndoh** [comment]
Yes sorry, my original ticket could use a bit more info couldn't it. I've done some more tests and here is a test case:
```csharp
using (var p = new ExcelPackage())
{
    var ws = p.Workbook.Worksheets.Add("Sheet1");
    ExcelComment comment = ws.Cells[1, 1].AddComment("My Comment", "Me");
    comment.Visible = true;
    ws.Cells[1, 1].IsRichText = true;
    ws.Cells[1, 1].Value = "test";
    p.SaveAs(new FileInfo(@"C:\temp\comment.xlsx"));
}
```
The cause appears to be the `ws.Cells[1, 1].IsRichText = true;` when its `true` the comments don't show up. Set it to `false` and the comments are back. In 5.5.5 the comments show up fine with rich text enabled.

Using the same code with version 5.5.5:

![555](https://user-images.githubusercontent.com/88682/117632073-cc7d0480-b174-11eb-9d7d-ac5c380794ed.png)

And version 5.6.3:

![563](https://user-images.githubusercontent.com/88682/117632026-c129d900-b174-11eb-98cf-c5ae2d6f95d9.png)

**Comment by JanKallman** [comment]
Ok, thanks, I'll have a look

**Comment by JanKallman** [comment]
Comments were incorrectly cleared when converting a cell to RichText. Fix added.

**Comment by JanKallman** [comment]
Fixed in 5.6.4
